runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -79,11 +79,22 @@ default boolean gameAlwaysOnTop()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "showUsernameInTitle",
+			name = "Show username in title",
+			description = "Show the last account you logged onto in the window title",
+			position = 5
+	)
+	default UsernameTitlePosition showUsernameInTitle()
```

Comment:
Just do this like OSB does, I think they do not have toggle, so I would not add one too.